x-pack/platform/plugins/shared/cases/server/saved_object_types/cases/schemas/v5.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+import { casesSchema as casesSchemaV4 } from './v4';
+
+export const casesSchema = casesSchemaV4.extends({
+  incremental_id: schema.maybe(schema.nullable(schema.number())), // Added to account for the .text mapping
```

Comment:
because you're extending v4 which extends v3 which has `incremential_id` you can make this schema:

```ts
// v5
casesSchemaV4.extends({ /* Same fields, just added new multi-type `.text` to increment_id */ })
```

Nothing bad should happen if you leave it, just not necessary to redeclare it.